glib/src/thread_guard.rs

```diff
@@ -111,4 +115,40 @@ impl<T> Drop for ThreadGuard<T> {
     }
 }
 
+impl<T: Default> Default for ThreadGuard<T> {
+    fn default() -> Self {
+        Self::new(T::default())
+    }
+}
+
 unsafe impl<T> Send for ThreadGuard<T> {}
+
+impl<T: ToValue + StaticType> ToValue for ThreadGuard<T> {
```

Comment:
You also want `impl<T: From<Value>> From<ThreadGuard<T>> for Value` or so

Reply:
done, hopefully right on the first try

Reply:
No